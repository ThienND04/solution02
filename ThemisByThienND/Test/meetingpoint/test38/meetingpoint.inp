2 2 7
1 2 20
2 1 20
5 3
8 9
7 3
8 9
8 9
1 4
9 3
