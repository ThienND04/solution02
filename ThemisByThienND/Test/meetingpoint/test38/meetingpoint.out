60
160
60
160
160
20
60
